services/app-api/src/domain-models/healthPlanPackage.ts

```diff
@@ -72,8 +72,6 @@ function packageSubmitters(pkg: HealthPlanPackageType): string[] {
 function convertContractToUnlockedHealthPlanPackage(
     contract: ContractType
 ): HealthPlanPackageType | Error {
-    console.info('Attempting to convert contract to health plan package')
-
```

Comment:
Nit: file organization - this function takes parameter from the rates refactor (`ContractType`) whereas everything else in this file is about the domain model HPP and takes HPP related arguments. Should be separate file in domain models.

Reply:
Agreed, I could see this being in `domain-models/contractAndRates` in some file that contains all the converter functions we will have. I think though I want to punt this to your ticket. I think you have to make converter functions for locked HPP.